py/server/deephaven/table_listener.py

```diff
@@ -509,23 +515,15 @@ def __init__(self, tables: Sequence[Table], listener: Union[Callable[[Dict[Table
         except Exception as e:
             raise DHError(e, "failed to create a merged listener adapter.") from e
 
-    def _process(self, replay_updates: Optional[jpy.JType] = None) -> None:
-        """Process the table updates from the listener recorders.
-
-        Args:
-            replay_updates (Optional[jpy.JType]): the replay updates, only provided during replay.
-        """
-        if replay_updates:
-            self.listener({t: TableUpdate(t, tu) for t, tu in zip(self.tables, j_list_to_list(replay_updates))}, True)
-        else:
-            self.listener({lr.table: lr.table_update() for lr in self.listener_recorders}, False)
+    def _process(self) -> None:
+        """Process the table updates from the listener recorders. """
+        self.listener({lr.table: lr.table_update() for lr in self.listener_recorders}, False)
 
-    def start(self, do_replay: bool = False, replay_lock: Literal["shared", "exclusive"] = "shared") -> None:
+    def start(self, do_replay: bool = False) -> None:
```

Comment:
This is a breaking change.  I think it is fine because we likely have no users that have twiddled the flag.  I marked the PR as breaking ... but the ticket description needs to describe what is breaking so that it can be documented in the release notes.  Also, the ticket needs to be changed to `feat!` or whatever the conventional commit details are for breaking changes.